server/game/player.js

```diff
@@ -195,6 +195,18 @@ class Player extends Spectator {
         return this.plotDiscard.size() + this.usedPlotsModifier;
     }
 
+    hasEnoughGold(gold) {
```

Comment:
This seems unnecessarily verbose, was 'if(foo >= bar)' really that hard to understand?

Reply:
It's not that it's hard to understand, it's the fact that `player.gold >= gold` becomes incorrect as soon as we introduce Iron Bank. This was an intermediate step. If you look at the Iron Bank PR, this method has a more complicated implementation that checks multiple gold sources.

Now, there may be merit in doing this:
```javascript
if(player.availableGold() >= gold)

// vs

if(player.hasEnoughGold(gold))
```

but it needs to be a method in both cases in order to take the optional parameters sent in the Iron Bank PR:

```javascript
if(player.availableGold({ playingType: 'marshal' }) >= gold)

// vs

if(player.hasEnoughGold(gold, { playingType: 'marshal' })
```

(after writing this I'm sort of leaning more toward the hypothetical `availableGold` above, but let me know your thoughts)